spec/controllers/idv/capture_doc_controller_spec.rb

```diff
@@ -138,6 +138,7 @@
         result = {
           step: 'capture_complete',
           flow_path: 'hybrid',
+          irs_reproofing: false,
```

Comment:
Do we have any specs where we can assert an expectation of this value being `true` ? If not, can we add one?

Reply:
@aduth  added.